azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/CSAgentPoolProfile.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.compute;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Attachable;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * An client-side representation for a container service agent pool.
+ */
+@Fluent
+public interface CSAgentPoolProfile extends
+    ChildResource<ContainerService>,
+    HasInner<ContainerServiceAgentPoolProfile> {
+
+    /**
+     * @return the number of agents (VMs) to host docker containers.
+     * Allowed values must be in the range of 1 to 100 (inclusive).

```

Comment:
we don't need to say what the allowed or default values are on the "getter" since this is R/O